VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -231,6 +233,7 @@ public void onClick(View v) {
             @Override
             public void onClick(View v) {
                 environment.getRouter().forceLogout(getActivity(), environment.getSegment(), environment.getNotificationDelegate());
+                SecurityUtil.clearUserData(MainApplication.application);
```

Comment:
This should be moved to the `forceLogout()` method implementation itself, rather than hardcoded inside a specific click listener.